src/styles/blocks/product-rating.scss

```diff
@@ -0,0 +1,31 @@
+.product-rating {
+  display: flex;
+  justify-content: space-between;
+
+  &__stars {
+    display: flex;
+  }
+
+  &__opinion {
+    margin: 0;
+  
+    font-size: $secondary-font-size;
+    font-weight: $secondary-font-weight;
+    line-height: 14px;
+  }
+}
+
+.stars {
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars--4 > :nth-child(-n+4) {
```

Comment:
Implenent your whole code for the stars. By that you make it easier to adjust the rate